Resources/Computing/ARCComputingElement.py

```diff
@@ -339,6 +339,12 @@ def getJobStatus( self, jobIDList ):
       gLogger.debug("ARC status for job %s is %s" % (jobID, arcState))
       if ( arcState ): # Meaning arcState is filled. Is this good python?
         resultDict[jobID] = self.mapStates[arcState]
```

Comment:
No? I want the resultDict of a particular job to be a given state (which is a string) and this dictionary is what is needed by the SiteDirector.

If I convert the value into a dict(), I am guessing the SiteDirector will barf. Have I misunderstood your comment here?